src/pages/Profile/Profile.jsx

```diff
@@ -0,0 +1,50 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import customInstance from "../../axios_http_client";
+
+const accessToken =
+  "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY2ZGYwNzEzYzAzZjRiNzMxZmZlM2MxMCIsImlhdCI6MTcyNTg5MjM3MiwiZXhwIjoxNzI4NDg0MzcyfQ.Ttw-YGZQQoDKX4zz_n-as0SDvld1gZJq4c9wffAJgFU";
```

Comment:
this token will be dynamic so we shouldn't hardcode it here

Reply:
This has been resolved now with the latest update, the token state is being managed by a useReducer and shared through the context, for access to all.